xbmc/pvr/channels/PVRChannelGroup.h

```diff
@@ -34,9 +34,6 @@ namespace EPG
 
 namespace PVR
 {
-#define PVR_INTERNAL_GROUP_ID_RADIO 1
-#define PVR_INTERNAL_GROUP_ID_TV    2
```

Comment:
also, what's the effect of dropping these for the channel manager dialog for radio channels?